android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -33,4 +44,199 @@ internal class CommonKtTests {
             setProxyURLString("this is not a url")
         }
     }
-}
\ No newline at end of file
+
+    private val mockApplicationContext = mockk<Application>(relaxed = true)
+    private val mockContext = mockk<Context>(relaxed = true)
+    private val mockPurchases = mockk<Purchases>()
+
+    @BeforeEach
+    fun setup() {
```

Comment:
I would add this `setup` at the top, since it will also be executed before the 3 tests that were already in this file

Reply:
agreed